senza/cli.py

```diff
@@ -581,17 +601,21 @@ def update(definition, region, version, parameter, disable_rollback, dry_run, fo
 @click.argument('version', callback=validate_version)
 @click.argument('parameter', nargs=-1)
 @region_option
+@parameter_file_option
 @json_output_option
 @click.option('-f', '--force', is_flag=True, help='Ignore failing validation checks')
-def print_cfjson(definition, region, version, parameter, output, force):
+def print_cfjson(definition, region, version, parameter, output, force, parameter_file):
     '''Print the generated Cloud Formation template'''
     region = get_region(region)
-    data = create_cf_template(definition, region, version, parameter, force)
+    data = create_cf_template(definition, region, version, parameter, force, parameter_file)
     print_json(data['TemplateBody'], output)
 
 
-def create_cf_template(definition, region, version, parameter, force):
+def create_cf_template(definition, region, version, parameter, force, parameter_file):
     region = get_region(region)
+    if parameter_file is not None:
```

Comment:
I would just do `if parameter_file:` to make it shorter (we can treat empty or other falsy values same as None here)